exercises/raindrops/test/test_raindrops.c

```diff
@@ -11,99 +12,106 @@ void tearDown(void)
 {
 }
 
+void test_convert_drops(int drops, char *expected)
+{
+   char *result = convert(drops);
+   TEST_ASSERT_EQUAL_STRING(expected, result);
+   free(result);
+}
+
 void test_one_yields_itself(void)
 {
-   TEST_ASSERT_EQUAL_STRING("1", convert(1));
+   test_convert_drops(1, "1");
 }
 
 void test_three_yields_pling(void)
 {
-   TEST_IGNORE();               // delete this line to run test
-   TEST_ASSERT_EQUAL_STRING("Pling", convert(3));
+   //TEST_IGNORE();               // delete this line to run test
```

Comment:
These need to remain _un_-commented in the test file here in the repository

Reply:
🤦‍♂️  👍